perl/lib/Sanger/CGP/PindelPostProcessing/FragmentFilterRules.pm

```diff
@@ -475,8 +475,10 @@ sub flag_019 {
   if($tum_geno[$previous_format_hash->{'FC'}] < 3){
     return $FAIL;
   }
-  # previous test confirms FC/FD can't be 0, so no div0 check required
-  if ($tum_geno[$previous_format_hash->{'FC'}] / $tum_geno[$previous_format_hash->{'FD'}] < 0.05){
+
+  my $tumfc_over_tumfd = $tum_geno[$previous_format_hash->{'FD'}] > 0 ? $tum_geno[$previous_format_hash->{'FC'}] / $tum_geno[$previous_format_hash->{'FD'}] : undef;
```

Comment:
I'd prefer the `undef` to be `-1`.  Clear it's not a valid value and will still fail.

Reply:
Mainly as you are doing a numeric comparison with the result.

Reply:
> I'd prefer the `undef` to be `-1`. Clear it's not a valid value and will still fail.

I will make the change with FF020 as the same check occurs there as well